pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -159,23 +194,31 @@ stdenv.mkDerivation rec {
     "-Donnxruntime_BUILD_UNIT_TESTS=ON"
     "-Donnxruntime_ENABLE_LTO=ON"
     "-Donnxruntime_USE_FULL_PROTOBUF=OFF"
+    (lib.cmakeBool "onnxruntime_USE_CUDA" cudaSupport)
+    (lib.cmakeBool "onnxruntime_USE_NCCL" cudaSupport)
   ] ++ lib.optionals pythonSupport [
     "-Donnxruntime_ENABLE_PYTHON=ON"
+  ] ++ lib.optionals cudaSupport [
+    "-DFETCHCONTENT_SOURCE_DIR_CUTLASS=${cutlass}"
+    "-Donnxruntime_CUDNN_HOME=${cudaPackages.cudnn}"
+    "-DCMAKE_CUDA_ARCHITECTURES=${cudaArchitecturesString}"
   ];
 
-  env = lib.optionalAttrs stdenv.cc.isClang {
+  env = lib.optionalAttrs effectiveStdenv.cc.isClang {
     NIX_CFLAGS_COMPILE = toString [
       "-Wno-error=deprecated-declarations"
       "-Wno-error=unused-but-set-variable"
     ];
   };
 
-  doCheck = true;
+  doCheck = !cudaSupport;
+
+requiredSystemFeatures = lib.optionals cudaSupport [ "big-parallel" ];
```

Comment:
Is this github or something happened to the indent?

Reply:
It was an automatically applied suggestion through the web UI, so that's probably why there's incorrect indentation.